app/assets/stylesheets/application.scss

```diff
@@ -272,3 +211,46 @@ a {
 .search-button:hover {
   background: rgba(0, 0, 0, 0.2);
 }
+
+.join-users {
+  background-color: $grey;
+  text-align: center;
+  padding: 20px;
+  color: $white;
+}
+
+.facebook {
+  background-color: $facebook-color;
+  font-family: "foundation-icons";
+  content: "\f1c4";
+  font-size: $icon-size;
+  line-height: 0.1rem;
+}
+
+.facebook:hover {
+  background-color: darken($facebook-color, 10%);
+}
+
+.twitter {
+  background-color: $twitter-color;
+  font-family: "foundation-icons";
```

Comment:
Properties should be ordered background-color, content, font-family, font-size, line-height